pom.xml

```diff
@@ -87,6 +88,32 @@
 			<artifactId>fhir-path</artifactId>
 			<version>${fhir.version}</version>
 		</dependency>
+
+		<dependency>
+			<groupId>org.projectlombok</groupId>
```

Comment:
Are we still using lombok on this lib?

Reply:
@rkodev are we suppose to remove this?

Reply:
let me confirm this will work on androiud

Reply:
Lombok works on android. We should be using it